app/javascript/Component/Greetings.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchMessage } from "../Redux/action";
+
+const GreetingsComponent = () => {
+  const dispatch = useDispatch();
+  const randomMessage = useSelector((state) => state.randomMessage);
+  console.log(randomMessage);
+
+  useEffect(() => {
+    // Dispatch the fetchMessage action when the component mounts
+    dispatch(fetchMessage());
+  }, [dispatch]);
+
+  console.log(randomMessage);
+
+  return (
+    <div>
+      <h2>Random Message:</h2>
+      <p>{randomMessage}</p>
+    </div>
```

Comment:
- [ ] Please make sure that your `application` is working as expected, it seems like your `random` greetings are not fetched properly form your API and you got `undefined` because you print the `randomMessage` at line number 15, because of which I was able to detect the issue from the console log, which you can also find at the below screenshot. Kindly fix this problem and make sure that your application is rendering the `random` greeting that is coming from your API.

![image](https://github.com/ismayito/hello-rails-react/assets/66476329/6ddf09dd-5103-405a-9488-93b78f11ef3b)